CHANGELOG.md

```diff
@@ -55,13 +56,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Retrieve UV flux arrays from Cloud-J used to set UV flux diagnostics
 - Fixed issue in `download_data.py` that was adding an extra `ExtData` to file paths
 - Restored `UVFlux` diagnostic output in the `fullchem_alldiags` integration test
+- Modified the carbon mechanism in KPP to separate tropospheric CH4 loss by OH from CO production by CH4 to remove dependency of CH4 and CO on each other and eliminate differences between CH4/tagCO simulations and the carbon simulation
+- Renamed several dummy species in the carbon mechanism for clarity
+- Fixed precision calculations within `co2_mod.F90` and `agged_co_mod.F90` to elimate differences with the carbon simulation
```

Comment:
Typo: "agged_co_mod" -> "tagged_co_mod"

Reply:
Typo: "elimate" -> "eliminate"

Reply:
Now fixed in https://github.com/geoschem/geos-chem/pull/2800/commits/f0d5265d97a7a91e275200c8c0b2f2d102ab085c